### ISSUE TITLE
[bug] Sync / Beacon does not generate QR codes

### ISSUE THREAD

**Issue by ktorn** [teia-community/teia-ui#247]
### Describe the bug

When trying to Sync with mobile wallets, or another device, the Beacon UI fails to generate the necessary QR code.

Console shows an error:

```
vendor-94e8067d.js:456 Uncaught (in promise) TypeError: qrcode is not a function
    at getQrData (vendor-94e8067d.js:456:15852)
    at vendor-94e8067d.js:459:58984
    at Generator.next (<anonymous>)
    at Jt (vendor-94e8067d.js:459:53541)
   ```
   
 Tried other platforms that use Beacon (objkt.com and fxhash) and it works, so it is not Beacon specific.

### Reproduction

1. Click the Sync button
2. Select any of the Mobile Wallet options, or "Pair wallet on another device" at the bottom
3. The dialog fails to load the expected QR code for syncing with the wallet.

### Expected behavior

Beacon dialog should display a QR code for syncing with the external wallet.

### Platform and versions

```shell
- OS: MacOS
- Browser: Firefox / Chrome
```

### Console output

```shell
vendor-94e8067d.js:456 Uncaught (in promise) TypeError: qrcode is not a function
    at getQrData (vendor-94e8067d.js:456:15852)
    at vendor-94e8067d.js:459:58984
    at Generator.next (<anonymous>)
    at Jt (vendor-94e8067d.js:459:53541)
   ```
```


### Additional context

_No response_

**Comment by melMass** [comment]
Thanks I will look into it later today!

**Comment by melMass** [comment]
Solved in 228db9c5a5467f9f7f1e7282752239465d0edb82
